source/code/include/playfab/PlayFabCallRequestContainer.h

```diff
@@ -15,11 +15,13 @@ namespace PlayFab
             const std::unordered_map<std::string, std::string>& headers,
             const std::string& requestBody,
             CallRequestContainerCallback callback,
-            void* customData = nullptr,
-            std::shared_ptr<PlayFabApiSettings> apiSettings = nullptr);
+            std::shared_ptr<PlayFabApiSettings> apiSettings,
+            std::shared_ptr<PlayFabAuthenticationContext> context,
+            void* customData);
 
         virtual ~CallRequestContainer() override;
         std::string GetFullUrl() const;
+        virtual bool ValidateSettings() override;
```

Comment:
>virtual [](start = 8, length = 7)

I don't think virtual is needed here

Reply:
You and Deb are absolutely right, this should not have been here. I was solving the immediate problem, but I forgot the purpose of the separation of BaseCRC (this one), and CRC, which inherits this one.
It's fixed now.

---
In reply to: [343375665](https://github.com/PlayFab/XPlatCoreTemplate/pull/39#discussion_r343375665) [](ancestors = 343375665)